modules/ui_extensions.py

```diff
@@ -325,15 +325,14 @@ def normalize_git_url(url):
     return url
 
 
-def github_proxy(url):
+def github_proxy_insteadof():
     proxy = shared.opts.github_proxy
-
     if proxy == 'None':
-        return url
-    if proxy == 'ghproxy.com':
-        return "https://ghproxy.com/" + url
+        return None
 
-    return url.replace('github.com', proxy)
+    if proxy == 'ghproxy.com':
+        return 'url.https://ghproxy.com/https://.insteadof=https://'

```

Comment:
ghproxy can only handle:
- github.com
- raw.githubusercontent.com
- gist.github.com
- gist.githubusercontent.com

A more specific rule may be desired

Reply:
you sure about that?
because it seems to work for me, and this is the only one that I actually can test as it is not blocked
![image](https://github.com/AUTOMATIC1111/stable-diffusion-webui/assets/40751091/05cfc6a4-6fbd-43a2-bb7f-e25fe2439cd2)

Reply:
do it this way: `'url.https://ghproxy.com/https://github.com/.insteadof=https://github.com/'`